doc/api/events.md

```diff
@@ -935,13 +935,351 @@ if the `EventEmitter` emits `'error'`. It removes all listeners when
 exiting the loop. The `value` returned by each iteration is an array
 composed of the emitted event arguments.
 
+## `EventTarget` and `Event` API
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+The global `EventTarget` and `Event` objects are a Node.js-specific adaptation
+of the [`EventTarget` Web API][] that are exposed by some Node.js core APIs.
+Neither the `EventTarget` nor `Event` classes are currently available for end
+user code to create.
+
+```js
+const target = getEventTargetSomehow();
+
+target.dispatchEvent({ type: 'foo' });
+
+target.addEventListener('foo', (event) => {
+  console.log('foo event happened!');
+});
+```
+
+The `EventTarget` class is similar in function to `EventEmitter` but evolved
+separately as part of browser DOM APIs. The Node.js implementation of
+`EventTarget` is an adaptation of the DOM API with a number of Node.js-specific
+extensions and behaviors intended to allow it to work more naturally within
+the Node.js environment.
+
+### Node.js `EventTarget` vs. DOM `EventTarget`
+
+There are four key differences between the Node.js `EventTarget` and the
+[`EventTarget` Web API][]:
+
+1. There is no concept of a hierarchical DOM and event propagation in Node.js.
+   That is, an event dispatched to an `EventTarget` does not propagate through
+   a hierarchy of nested target objects that may each have their own set of
+   handlers for the event.
```

Comment:
This is not really a difference. There are plenty of EventTargets for which "get the parent" returns null on the web platform, such as those returned by `new EventTarget()`.

Reply:
The point remains, the fact that there is no concept of hierarchy *at all* in Node.js (as opposed to hierarchy being intrinsic in the DOM and *only some* `EventTargets` not needing it means that we don't need any of the propagation logic, properties, etc.

Reply:
I think you're phrasing it backward. Hierarchy is optional in the DOM. So if Node doesn't have hierarchical objects, then it's no different than the DOM. Both environments have hierarchical EventTarget behavior for all of the hierarchical objects that exist.